@navikt/core/react/src/form/combobox/SelectedOptions/selectedOptionsContext.tsx

```diff
@@ -101,13 +101,16 @@ const SelectedOptionsProvider = ({
     [customOptions, onToggleSelected, removeCustomOption],
   );
 
-  const isLimitReached =
-    (!!maxSelected?.limit && selectedOptions.length >= maxSelected.limit) ||
-    (!isMultiSelect && selectedOptions.length > 0);
+  const isLimitReached = useMemo(() => {
+    return (
+      (!!maxSelected?.limit && selectedOptions.length >= maxSelected.limit) ||
+      (!isMultiSelect && selectedOptions.length > 0)
+    );
+  }, [maxSelected, selectedOptions, isMultiSelect]);
```

Comment:
Its memozied to avoid this useEffect from running all the time: 
```
useEffect(() => {
    setHideCaret(isLimitReached);
  }, [isLimitReached, setHideCaret]);
```

Reply:
Since `isLimitReached` is a boolean, it will not change on each render, as opposed to an object.

Reply:
🧠 🧠 🧠 🧠 🧠 🧠 🧠 TIL! So used to using bools in states like `const [flag, setFlag]= useState..` where changing `flag` to the same value will cause useEffect from running that i didn't even know this